CMakeLists.txt

```diff
@@ -216,6 +216,11 @@ elseif((CMAKE_CXX_COMPILER_ID STREQUAL "Clang") OR (CMAKE_CXX_COMPILER_ID STREQU
 endif()
 target_link_libraries(kvrocks_objs PUBLIC -fno-omit-frame-pointer)
 target_link_libraries(kvrocks_objs PUBLIC ${EXTERNAL_LIBS})
+if (ENABLE_SPEEDB)
+    target_compile_definitions(kvrocks_objs PUBLIC SPEEDB_STORAGE_ENGINE)
```

Comment:
Could we just use one macro definition with different value?

Reply:
Thank you for the feedback. I'm thinking of having numbered values for each storage engine type. Would that be ideal?
or we could have string names like `"ROCKSDB"` and `"SPEEDB"`.

- First case:

`STORAGE_ENGINE=0 # for rocksdb`
`STORAGE_ENGINE=1 # for speedb`

- Second case:

`STORAGE_ENGINE="ROCKSDB"`
`STORAGE_ENGINE="SPEEDB"`

Reply:
I vote for the second one